syft/pkg/cataloger/binary/pe_package.go

```diff
@@ -0,0 +1,134 @@
+package binary
+
+import (
+	"path"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/anchore/syft/syft/file"
+	"github.com/anchore/syft/syft/pkg"
+)
+
+var (
+	// spaceRegex includes nbsp (#160) considered to be a space character
+	spaceRegex  = regexp.MustCompile(`[\s\xa0]+`)
```

Comment:
smallest nit: I'm sure we have this elsewhere and probably should be using this elsewhere -- it could be useful to have some standard functions/regexes/etc. that catalogers could share rather than re-writing this multiple times; and `#160` isn't always obvious to include in whitespace, but we probably want to almost everywhere we are trimming or otherwise dealing with whitespace. Maybe a shared `spaceNormalize` is what I'm thinking of

Reply:
I don't see any other places in the code where we're doing `ReplaceAll(subject, <space-like-things>, " ")` (or similar) --if you find some examples of this happy to refactor in a follow up PR.